### DISCUSSION TITLE
Show a warning on state machine execution

### DISCUSSION THREAD

**Discussion by franzlst** [DLR-RM/RAFCON#938] [Ideas]
Show a warning if a library state was changed and another state machine, which includes the library, is started without reloading.

Originally created by @sebastian-brunner  (Sebastian.Brunner@dlr.de) at 2017-05-09 12:03:33+00:00 (moved from RMC internal repository)

**Comment by JohannesErnst** [comment]
I think this is a very good idea and need feature
